vendor-extra/JatsContentBundle/src/EventListener/BodyListener.php

```diff
@@ -23,9 +23,13 @@ public function __construct(ViewConverter $converter)
         $this->converter = $converter;
     }
 
-    public function onCreatePageMain(CreateContentPagePartEvent $event) : void
+    public function onCreatePagePart(CreatePagePartEvent $event) : void
     {
-        $body = $event->getItem()->xpath()->firstOf(self::BODY_PATH);
+        if ('content' !== $event->getRequest()->attributes->get('libero_page')['type']) {
```

Comment:
could the `CreatePagePartEvent` class host more logic from these checks, like a `isForContent()` or `isFor($type)` method?

Reply:
See also https://github.com/libero/browser/pull/52#discussion_r266850298.